base_user_role/data/ir_cron.xml

```diff
@@ -3,16 +3,17 @@
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <odoo noupdate="1">
 
-<record model="ir.cron" id="cron_update_users">
-    <field name='name'>Update user roles</field>
-    <field name='interval_number'>3</field>
-    <field name='interval_type'>hours</field>
-    <field name="numbercall">-1</field>
-    <field name="active">True</field>
-    <field name="doall" eval="False" />
-    <field name="model">res.users.role</field>
-    <field name="function">cron_update_users</field>
-    <field name="args">()</field>
-</record>
+    <record model="ir.cron" id="cron_update_users">
+        <field name='name'>Update user roles</field>
+        <field name='interval_number'>3</field>
+        <field name='interval_type'>hours</field>
+        <field name="numbercall">-1</field>
+        <field name="active">True</field>
+        <field name="doall" eval="False"/>
+        <field name="model_id" ref="base_user_role.model_res_users_role"/>
+        <field name="state">code</field>
+        <field name="code">model.cron_update_users()</field>
+        <field name="args">()</field>
+    </record>
```

Comment:
Is it worth it to reindent all the XML (I take the OCA template module as the reference: https://github.com/OCA/maintainer-tools/blob/master/template/module/data/model_name_data.xml)? It makes big diff for no value, and will make harder to backport&merge code from older branches.

Reply:
I think you're right, thanks for you comment :+1:

Reply:
@novawish it seems that the XML ./data/ files should be fixed as the view ones

Reply:
Done! I also squashed all commits